src/app/controllers/CartController.ts

```diff
@@ -156,15 +157,17 @@ export class CartController {
    * Remove an item from currently active cart.
    * @param {string} cartId
    * @param {string} cartItemId
+   * @param {string} sharedToken
```

Comment:
This should be optional. Also, do we even need the jsdocs since theres no description provided and we already know the types because TS.